### PR TITLE
Display 'Reply' in case of single replies

### PR DESCRIPTION
**Pull request by VibhorCodecianGupta**
Resolves issue 12
This now displays 'Reply' as opposed to earlier 'Replies' for single reply threads

### CODE REVIEW

File: app/pods/components/doubt-view/component.js [hunk 1]
```diff
@@ -21,9 +21,14 @@ export default class DoubtViewComponent extends Component {
     return env.discussBaseUrl + '/' + avatar_template.replace('{size}', '150')
   }
 
+  @computed('topicResponse.posts_count')
+  get numberOfPosts () {
```
Comment (abhishek97) [inline app/pods/components/doubt-view/component.js]: This is misleading. I would rather prefer a computed property which is a boolean as to whether the number of posts is `>1` or not.

An alternate way is to use https://github.com/jmurphyau/ember-truth-helpers (it already used in this project) and use `gt` helper in the template itself.

**Comment by abhishek97** [comment]
@VibhorCodecianGupta Please update this PR.

**Comment by VibhorCodecianGupta** [comment]
@abhishek97 @witty123 updated PR, please have a look

File: app/pods/components/doubt-view/template.hbs [hunk 2]
```diff
@@ -10,7 +10,11 @@
         <span class="time">{{moment-from-now topic.created_at}}</span>
       </div>
     </div>
+    {{#if (eq topicResponse.posts_count 1)}}
+    <div class="text"> {{topicResponse.posts_count}} Reply</div>
```
Comment (abhishek97) [inline app/pods/components/doubt-view/template.hbs]: move the `{{#if}}{{/if}}` block inside the div.

**Comment by VibhorCodecianGupta** [comment]
@abhishek97 updated, please review

**Comment by VibhorCodecianGupta** [comment]
@witty123 @abhishek97 please review

**Pull request merged**